rules/docker_config.bzl

```diff
@@ -256,14 +256,29 @@ def _docker_toolchain_autoconfig_impl(ctx):
         output_tarball = image_tar,
         workdir = bazel_config_dir,
     )
-
+    
+    # Commands to run script to create autoconf results, output stderr to log file
+    # add the log file to a tar file and append the output.tar to that same tar file
+    commands = []
+    commands+= ["/" + ctx.attr.name + "_install.sh 2> /" + ctx.attr.name + "_log.log"]
+    commands+= ["tar -cf /extract.tar /" + ctx.attr.name + "_log.log"]
+    commands+= ["tar -rf /extract.tar /" + ctx.attr.name + "_outputs.tar"]
```

Comment:
Combine line 264 and 265 in one command?

Reply:
I split them to be able to deal with errors in case outputs.tar was not produced. Updated code to include proper handling of error case (i.e., no failure is produced in extract rule if outputs.tar is not produced)